docs/source/getting_started/03_custom_nodes.md

```diff
@@ -113,7 +106,7 @@ from peekingduck.pipeline.nodes.node import AbstractNode
 from .utils.csv import CSVLogger
 
 class Node(AbstractNode):
-    """Node that draws object counting"""
+    """Node that logs outputs of PeekingDuck and writes to a CSV"""
```

Comment:
- For init, don't need to get the configs from `config` anymore. Can use self.filepath directly (see updated nodes for example)
- Both this snippet and the one above it needs to be updated with **kwargs etc
```
    def __init__(self, config: Dict[str, Any] = None, **kwargs: Any) -> None:
        super().__init__(config, node_path=__name__, **kwargs)
        self.model = posenet_model.PoseNetModel(self.config)

    def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
```
- So does the node_template.py file https://github.com/aimakerspace/PeekingDuck/blob/dev/peekingduck/pipeline/nodes/node_template.py

Reply:
updated. node_template.py should be updated in another issue